etna/models/nn/utils.py

```diff
@@ -310,3 +312,34 @@ def _make_target_prediction(self, ts: TSDataset, horizon: int) -> Tuple[TSDatase
         ts.df = ts.df.iloc[-horizon:]
         ts.loc[:, pd.IndexSlice[:, "target"]] = predicts.T[:horizon]
         return ts, prediction_dataloader
+
+
+class MultiEmbedding(nn.Module):
+    """Class for obtaining the embeddings of the categorical features."""
+
+    def __init__(self, embedding_sizes: Dict[str, Tuple[int, int]]):
+        """
+        Init MultiEmbedding.
+
+        Parameters
+        ----------
+        embedding_sizes:
+            dictionary mapping feature name to tuple of number of categorical classes and embedding size
+        """
+        super().__init__()
+
+        self.embedding_sizes = embedding_sizes
+        self.embedding = nn.ModuleDict(
+            {feature: nn.Embedding(n + 1, dim) for feature, (n, dim) in self.embedding_sizes.items()}
```

Comment:
Why n+1?

Reply:
We have discussed that we need one more embedding for unknown category

Reply:
We should clarify it using the comment and may be write about this in documentation of the class.

Reply:
I think we don't need to write about deep implementation nuances in documentation of the class.
I will write comment  above this line